doc/presentation/start_presentation.txt

```diff
@@ -12,7 +12,10 @@ OSGeoLive
 
     .. rv_small::
 
-        Press "s" for slide notes, ESC to zoom out, SPACE / arrow keys to navigate.
+        Press ESC to zoom out, SPACE / arrow keys to navigate, s for slide notes.
```

Comment:
We need to say "the s key". Perhaps something like "...navigate, and the S key to view the slide notes."

Reply:
because this is a presentation, with limited chars, I've been deliberately concise in word choice and dropping words.
It this case, I won't take on the extra words. (But that for picking up on the capitalisation.)